### PR TITLE
search bar styling update

### LINKED ISSUE #30: Search bar styling update
**Is your feature request related to a problem? Please describe.**
Fixing the search bar css 

**Describe the solution you'd like**
Update animation transition and icon positioning

### PR DESCRIPTION
**Pull request by gonzalespaulb**
## Changes
_List Changes Introduced by this PR_
1. Restyled the search bar
2. Integrated the old logic with the new input 

## Purpose
To fix the styling issues from the first iteration of the search bar

_If this closes an issue, name it here. If it doesn't, remove this line_
Closes #30

### CODE REVIEW

File: movie-app/src/header/NavBar.js [hunk 1]
```diff
@@ -47,12 +47,16 @@ const NavBar = ({ pageSet, movieSearch, setMovieSearch }) => {
...
+        <form id="search-form" onSubmit={(e) => {
+          e.preventDefault(); 
+          titleChecker(e.target[0].value);
+        }}>
           <div 
             onClick={openSwitch} 
             className={applyStyles(`btn`)}
           >
             <img 
+              //Get the onClick to work
```
Comment (connielion) [inline movie-app/src/header/NavBar.js]: This comment should not be merged to development.

File: movie-app/src/header/NavBar.js [hunk 2]
```diff
@@ -1,76 +1,93 @@
...
+  const [cherryPopped, setCherryPopped] = useState(false);
 
   const clearText = () => {
     document.getElementById("input").value = "";
   };
 
-  const titleChecker = (e) => {
-    e.preventDefault();
-    if (e.target[0].value.length < 3) {
+  //Get title checker to be able to be fired from the search button and entered with input
```
Comment (connielion) [inline movie-app/src/header/NavBar.js]: Is this comment supposed to be here?

File: movie-app/src/header/NavBar.js [hunk 3]
```diff
@@ -1,76 +1,93 @@
 import { useState } from "react";
+import cancel from '../assets/search-cancel.svg';
+import search from '../assets/search-magnify.svg';
 
-const NavBar = ({ movieSearch, setMovieSearch }) => {
-  const [searchOpen, setSearchOpen] = useState();
+const NavBar = ({ pageSet, movieSearch, setMovieSearch }) => {
+  const [active, setActive] = useState(false);
+  const [cherryPopped, setCherryPopped] = useState(false);
```
Comment (krferrell) [inline movie-app/src/header/NavBar.js:7]: cherryPopped seems like a slightly inappropriate name.

**Review by krferrell — DISMISSED** [review]

**Review by connielion — APPROVED** [review]
Looks good.

**Pull request merged**